.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build @extrahorizon/javascript-sdk
+
+on:
+  push:
+    branches-ignore:
+      - dev
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Create .npmrc
+        run: |
+          echo "@extrahorizon:registry=https://npm.pkg.github.com" > .npmrc
+          echo "//npm.pkg.github.com/:_authToken=${{secrets.GITHUB_TOKEN}}" >> .npmrc
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://npm.pkg.github.com/
+          scope: '@extrahorizon'
+      - run: yarn
+      - run: yarn lint
+      - run: yarn run prettier --check
+      - run: yarn build
+      - run: yarn test
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - uses: paambaati/codeclimate-action@v2.7.5
```

Comment:
Copied from quality-assurance.yaml, not sure if we want to keep this. Will add to question document.